AutomatedTesting/Gem/PythonTests/Atom/TestSuite_Main_Null_Render_Level_Loads.py

```diff
@@ -11,7 +11,7 @@
 from ly_test_tools.o3de.editor_test import EditorBatchedTest, EditorTestSuite
 
 logger = logging.getLogger(__name__)
-TEST_DIRECTORY = os.path.join(os.path.dirname(__file__), "tests")
+TEST_DIRECTORY = os.path.join(os.path.dirname(__file__), "tests").lower()
```

Comment:
Same issue as above

Reply:
these aren't even needed any more nor is `logger` so just going to delete those completely